### ISSUE TITLE
Warn nominators if they are below the stake threshold

### ISSUE THREAD

**Issue by kianenigma** [polkadot-js/apps#5632]
https://github.com/paritytech/substrate/pull/8920 should ideally be reflected in the UI in a few ways:

1. when you bond, if your bonded amount is less than both `MinNominatorBond ` or `MinValidatorBond`, you should be warned that you will not be allowed to participate.
2. if you are already bonded and you are below either of the thresholds, you should be shown a warning that you might be chilled via the new permission-less chill extrinsic.
3. Since the new limits are now static and governance controlled, they can be simply shown in the apps (as opposed to something like https://polkaview.network/dot/staking which looks at the dynamic limits).

Since this affects a lot of people, it should preferably be treated as priority.

**Comment by michalisFr** [comment]
Proposal: Add the following message on "Account actions" and "Targets" pages generally.

```
Currently only the top 20,000 nominators (or whatever the number may be in the future), in terms of staked balance, can participate in the nomination process. The current minimum staked balance to be in the top 20,000 is XXX DOT. This minimum balance changes dynamically every era (every 24 hours).

You can stake your funds, but if you are not in the top 20,000 you will not be able to nominate. If at any time you find yourself below the current minimum balance, you may automatically stop nominating. 

Remember that unstaking your DOT takes 28 days.
```

The `current minimum staked balance` should be the lower of the hard cap and the soft cap (speaking in layman's terms)

**Comment by jacogr** [comment]
Sorry missed the issue - closed in https://github.com/polkadot-js/apps/pull/5674
